src/main/scala/collections/dataAnalysis/Loader.scala

```diff
@@ -0,0 +1,27 @@
+package collections.dataAnalysis
+
+import scala.collection.View
+import scala.io.Source
+import scala.util.{Try, Using}
+
+case class Loader() {
```

Comment:
I don't see any reason to make a Loader case class, should be object

Reply:
Changed to object